### PR TITLE
Make krel promote-images accept more than one tag

### PR DESCRIPTION
**Pull request by puerco**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:

This PR modifies the `krel promote-images` command to take more than one `--tag` to patch the manifest with images with more than one tag.

#### Which issue(s) this PR fixes:

Ref: https://kubernetes.slack.com/archives/CJH2GBF7Y/p1602695125217400?thread_ts=1602694864.214600&cid=CJH2GBF7Y

#### Does this PR introduce a user-facing change?

```release-note
- krel promote-images will now accept more than one --tag when patching the promotion manifest
```

### CODE REVIEW

File: cmd/krel/cmd/promote-images.go [hunk 1]
```diff
@@ -243,7 +248,7 @@ func runPromote(opts *promoteOptions) error {
 		return errors.Wrap(err, "adding image manifest to staging area")
 	}
 
-	commitMessage := fmt.Sprintf("releng: Image promotion for %s", opts.tag)
+	commitMessage := fmt.Sprintf("releng: Image promotion for %s", opts.tags[0])
```
Comment (cpanato) [inline cmd/krel/cmd/promote-images.go:251]: maybe iterate and add all tags?
like `releng: Image promotion for 1.2.3 / 1.3.4`

just a suggestion not blocking at all

Comment (puerco) [reply cmd/krel/cmd/promote-images.go]: Good idea :) Done :+1:

**Review by cpanato — APPROVED** [review]
thanks!! :) 
/lgtm

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by puerco** [comment]
Patch releases are out.
/hold cancel

**Pull request merged**